chapter-10-workload-identities/4-deploy-api-gateway.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#############################
+# Deploy the Kong API gateway with a 
```

Comment:
incomplete sentence. `with an external certificate`?

Reply:
Ah! `with a sidecar"? or `with a service mesh`?

Reply:
Fixed - well spotted - updated to this:

# Deploy the Kong API gateway and add a service mesh sidecar to the gateway pods
# This enables the gateway pods to initiate mTLS connections to upstream workloads